### PR TITLE
config: accelsao is now jayzhan211

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### CODE REVIEW

File: teams/tikv/team.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
         "Fullstop000",
         "Jibbow",
         "Xuanwo",
-        "accelsao",
+        "jayzhan211",
```
Comment (BusyJay) [inline teams/tikv/team.json:56]: What if other users change name and don't inform us? I worry that the file is critical to permission and if others reclaim the user name then can do harm to all the projects.

**Comment by BusyJay** [comment]
Instead of writing all the names here, I suggest to use public github team url.

**Comment by tisonkun** [comment]
> team

@BusyJay Sounds reasonable. Given we create these teams already, we can refer there as single source of truth.

**Comment by tisonkun** [comment]
closing as prefer SSOT

**Comment by tisonkun** [comment]
@BusyJay though the GitHub team url is invisible to people outside the org, but we can host a people page outsider.

**Comment by BusyJay** [comment]
They can be changed: https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility. Not all teams need to be public, just those included inside these files.

**Comment by tisonkun** [comment]
> They can be changed: https://docs.github.com/en/organizations/organizing-members-into-teams/changing-team-visibility. Not all teams need to be public, just those included inside these files.

Note that the most public level is "can be seen and mentioned by every **member of this organization**."

**Comment by tisonkun** [comment]
> I worry that the file is critical to permission

@BusyJay Nope. It is for display. The permission is controlled by team settings. The settings are SSOT and we can always update this display file.

**Comment by tisonkun** [comment]
Reopen since we still use this file for metadata handling.

**Comment by tisonkun** [comment]
@hi-rustin any suggestion on this issue?

**Comment by hi-rustin** [comment]
> @hi-rustin any suggestion on this issue?

I've retired from most community related things, so @Mini256 could you please take a look?

**Comment by Mini256** [comment]
> @hi-rustin any suggestion on this issue?

Perhaps we use people files to store all contributor metadata contain gitHub login and github id, as the Rust Community does (https://github.com/rust-lang/team/tree/master/people).

When `team.json` is modified, we can check if the newly added Github login is registered with the `people` directory via CI, and if there is no registration, it refuses to modify it. 

The metadata in the `people` directory can be generated by the CLI tool.

Check whether the metadata in the people directory is correct regularly through the program. If not, inform the maintainer to fix  it.

**Comment by tisonkun** [comment]
@BusyJay I'd suggest to merge this pull request for now to correct the information. Anyway it is a low traffic action.

I'll try to investigate a proper way to handle membership later.

**Review by BusyJay — APPROVED** [review]

**Comment by tisonkun** [comment]
assign @BusyJay

**Pull request merged**